chain/rust/src/fees.rs

```diff
@@ -50,6 +62,67 @@ pub fn min_script_fee(
     }
 }
 
+
+
+// pub fn total_ref_scripts_size(ref_inputs: &[TransactionUnspentOutput]) -> Result<u64, ArithmeticError> {
+//     ref_inputs
+//         .iter()
+//         .try_fold(0, |acc, utxo|
+//             utxo
+//                 .output
+//                 .script_ref()
+//                 .map(|script| script.to_cbor_bytes().len() as u64)
+//                 .unwrap_or(0u64)
+//                 .checked_add(acc)
+//                 .ok_or(ArithmeticError::IntegerOverflow))
+// }
+
+/**
+ * Calculates the cost of all ref scripts
+ * * `total_ref_script_size` - Total size (original, not hashes) of all ref scripts. Duplicate scripts are counted as many times as they occur
+ */
+pub fn min_ref_script_fee(
+    tx: &Transaction,
+    linear_fee: &LinearFee,
+    total_ref_script_size: u64,
```

Comment:
It isn't possible to know the original sizes so we can't compute this from just the `Transaction` anymore